src/datamigration/azext_datamigration/manual/helper.py

```diff
@@ -36,6 +56,27 @@ def validate_os_env():
         raise CLIInternalError("This command cannot be run in non-windows environment. Please run this command in Windows environment")
 
 
+# -----------------------------------------------------------------------------------------------------------------
+# Helper function to check if the input string is in valid Guid format or not
+# -----------------------------------------------------------------------------------------------------------------
+def is_valid_guid(guid):
+    try:
+        _ = uuid.UUID(guid)
+    except ValueError:
+        return False
+    return True
+
+
+# -----------------------------------------------------------------------------------------------------------------
+# Helper function to check if the input string is in valid base64 string or not
+# -----------------------------------------------------------------------------------------------------------------
+def is_base64(s):
+    try:
+        return base64.b64encode(base64.b64decode(s)).decode() == f'{s}'
```

Comment:
nit: Is f'{s}' same as s? Can we replace?

Reply:
Done, rightly called. f'{s}' is not required.